OpenRA.Mods.Common/Widgets/Logic/TabCompletionLogic.cs

```diff
@@ -20,60 +20,91 @@ public class TabCompletionLogic : ChromeLogic
 		IList<string> candidates = new List<string>();
 		int currentCandidateIndex = 0;
 		string lastCompleted;
+		public readonly string PrefixOnStart;
+		public readonly string SuffixOnStart;
 		string prefix;
 		string suffix;
 
-		public IList<string> Commands { get; set; }
+		int longestElementLength;
+		IList<string> universe;
+		public IList<string> Universe
```

Comment:
This is potentially confusing here, since we are simulating `Actor`s, and `World`s.  Long ago we were going to have a `Universe` class that wrapped the concept of the world and other bits.